### PR TITLE
CSS

### PR DESCRIPTION
**Pull request by smartfuse**

### CODE REVIEW

File: app/assets/html/survey_questions.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div id="survey-questions-container">
     <div class="header">{{ currentTopic }}</div>
-    <p id="description">{{ pageDescription() }}</p>
+    <p id="description">{{ numQuestions }} questions.</p>
```
Comment (leohtkam) [inline app/assets/html/survey_questions.html:3]: Change one of the conditional in pageDescription method in surveyQuestions.coffee instead. What you did here will completely break state knowledge.

File: features/state_knowledge.feature [hunk 2]
```diff
@@ -32,13 +32,3 @@ Scenario: Correct message is displayed when going back to topic selection page
     And I press "Continue to Survey Questions"
     And I press "Back"
     Then I should see "Edit selections and press Next to continue"
-
```
Comment (leohtkam) [inline features/state_knowledge.feature:35]: -_- You shouldn't delete a scenario to pass all the tests. See my comment above on how to fix this.

**Comment by leohtkam** [comment]
Please fix the state knowledge feature that is broken by this.

**Comment by impguard** [comment]
-1. Until you rebase appropriately on iteration3, it's really screwing with git. I suggest making a fresh new branch - call it something different if it's easier - such as "iteration3-css". Make sure this branch is based off of the current iteration3 branch, and apply your less fixes there. I've tried manually cherry-picking your changes into this branch on my own locally, but the conflicts are too difficult for me to parse.

**Pull request closed without merging**